### PR TITLE
Ddpb 1734 add user casrec fixtures

### PR DESCRIPTION
**Pull request by slizzio**

### CODE REVIEW

File: src/AppBundle/Command/AddSingleUserCommand.php [hunk 1]
```diff
@@ -94,6 +96,11 @@ protected function addSingleUser(OutputInterface $output, array $data, array $op
 
         $user->setPassword($this->encodePassword($user, $data['password']));
 
+        if ($data['roleName'] != User::ROLE_ADMIN) {
```
Comment (elvisciotti) [inline src/AppBundle/Command/AddSingleUserCommand.php]: @slizzio we discussed adding the client here, as the idea behind the fixture is pre-creating the data

Comment (slizzio) [reply src/AppBundle/Command/AddSingleUserCommand.php]: Agreed to do this post merge as users are needed to QA registration steps, Ive created a new ticket for this DDPB-1837

File: docker/confd/templates/parameters.yml.tmpl [hunk 2]
```diff
@@ -22,8 +21,15 @@ parameters:
...
             password: {{ getv (printf "/fixtures/%s/password" .) }}
             roleId: {{ getv (printf "/fixtures/%s/roleid" .) }}
+            roleName: {{ getv (printf "/fixtures/%s/rolename" .) }}
             activated: {{ getv (printf "/fixtures/%s/activated" .) }}
             odrEnabled: {{ getv (printf "/fixtures/%s/odr" .) }}
+            {{ if exists (printf "/fixtures/%s/deputyno" . ) }}deputyNo: {{ getv (printf "/fixtures/%s/deputyno" .) }}{{ end }}
+            {{ if exists (printf "/fixtures/%s/deputypostcode" . ) }}deputyPostcode: {{ getv (printf "/fixtures/%s/deputypostcode" .) }}{{ end }}
+            {{ if exists (printf "/fixtures/%s/casenumber" . ) }}caseNumber: {{ getv (printf "/fixtures/%s/casenumber" .) }}{{ end }}
+            {{ if exists (printf "/fixtures/%s/clientsurname" . ) }}clientSurname: {{ getv (printf "/fixtures/%s/clientsurname" .) }}{{ end }}
+            typeOfReport: {{ if exists (printf "/fixtures/%s/typeofreport" . ) }}{{ getv (printf "/fixtures/%s/typeofreport" .) }}{{ end }}
```
Comment (elvisciotti) [inline docker/confd/templates/parameters.yml.tmpl]: @slizzio  typeOfReport needs to go inside the "if"

File: docker/confd/templates/parameters.yml.tmpl [hunk 3]
```diff
@@ -22,8 +21,15 @@ parameters:
             lastname: {{ getv (printf "/fixtures/%s/lastname" .) }}
             password: {{ getv (printf "/fixtures/%s/password" .) }}
             roleId: {{ getv (printf "/fixtures/%s/roleid" .) }}
+            roleName: {{ getv (printf "/fixtures/%s/rolename" .) }}
```
Comment (elvisciotti) [inline docker/confd/templates/parameters.yml.tmpl]: we might not need this, but until it's not all merged, I think we need to wrap this with an "if" as well, not sure about the merge order atm

**Pull request merged**